src/main.cpp

```diff
@@ -46,7 +46,7 @@ void usage()
   std::cerr << "    BPFTRACE_NO_CPP_DEMANGLE  [default: 0] disable C++ symbol demangling" << std::endl;
   std::cerr << "    BPFTRACE_MAP_KEYS_MAX     [default: 4096] max keys in a map" << std::endl;
   std::cerr << "    BPFTRACE_CAT_BYTES_MAX    [default: 10k] maximum bytes read by cat builtin" << std::endl;
-  std::cerr << "    BPFTRACE_MAX_PROBES       [default: 512] max number of probes bpftrace can attach to" << std::endl;
+  std::cerr << "    BPFTRACE_MAX_PROBES       [default: 512] max number of probes bpftrace" << std::endl;
```

Comment:
"probes bpftrace" reads a little funny. How about this?
```suggestion
  std::cerr << "    BPFTRACE_MAX_PROBES       [default: 512] max number of attached probes" << std::endl;
```

Reply:
thanks, I missed a word. Meant to just say "max number of probes". I don't want to use the term "attached", as that's a bcc-thing where attaching is a separate action. Pushed an update.